### PR TITLE
Finish first task

### PR DESCRIPTION
**Pull request by jenyafomin**

### CODE REVIEW

**Comment by jenyafomin** [comment]
.

File: contracts/Vesting/AbstractVesting.sol [hunk 1]
```diff
@@ -12,7 +12,7 @@ contract AbstractVesting is VestingWallet, Ownable, Multicall {
     address beneficiaryAddress,
     uint64 startTimestamp,
     uint64 durationSeconds
-  ) VestingWallet(address(1), startTimestamp, durationSeconds) {
+  ) VestingWallet(address(beneficiaryAddress), startTimestamp, durationSeconds) {
```
Comment (TrejGun) [inline contracts/Vesting/AbstractVesting.sol:15]: this is wrong. 
address(1) is used to disable original beneficiaryAddress

File: contracts/Vesting/AbstractVesting.sol [hunk 2]
```diff
@@ -23,12 +23,14 @@ contract AbstractVesting is VestingWallet, Ownable, Multicall {
 
   function releaseable() public view virtual returns (uint256) {
     return vestedAmount(uint64(block.timestamp)) - released();
+
```
Comment (TrejGun) [inline contracts/Vesting/AbstractVesting.sol:26]: ??

File: contracts/Vesting/AbstractVesting.sol [hunk 3]
```diff
@@ -23,12 +23,14 @@ contract AbstractVesting is VestingWallet, Ownable, Multicall {
...
   function releaseable() public view virtual returns (uint256) {
     return vestedAmount(uint64(block.timestamp)) - released();
+    
   }
 
   function releaseable(address token) public view virtual returns (uint256) {
     return vestedAmount(token, uint64(block.timestamp)) - released(token);
   }
 
+  // ! Не понял с какой целью будет использоваться.
```
Comment (TrejGun) [inline contracts/Vesting/AbstractVesting.sol:33]: all communication in GH should be in english

File: hardhat.config.ts [hunk 4]
```diff
@@ -19,6 +19,9 @@ export default {
       allowUnlimitedContractSize: true,
     },
   },
+  // paths:{
```
Comment (TrejGun) [inline hardhat.config.ts:22]: remove

File: test/Vesting/transferOwnership.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      const receiver = (await ethers.getSigners())[1];
+      const { contractInstance } = await deployVestingFixture("GradedVesting");
+      // ethers.constants.AddressZero
+      const tx = contractInstance.connect(receiver).transferOwnership(ethers.constants.AddressZero);
+      await expect(tx).to.be.revertedWith("Ownable: new owner is the zero address");
+    });
+  });
+}
+
+shouldTransferOwnership();
```
Comment (TrejGun) [inline test/Vesting/transferOwnership.ts:33]: ??

File: test/Vesting/owner.ts [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    it("Should set the right roles to deployer", async function () {
+      const [_owner, receiver] = await ethers.getSigners();
+      const { contractInstance } = await deployVestingFixture("GradedVesting");
+
+      const owner = await contractInstance.owner();
+      expect(owner).to.equal(receiver.address);
+    });
+  });
+}
+shouldHaveOwner();
```
Comment (TrejGun) [inline test/Vesting/owner.ts:16]: ???

**Review by TrejGun — CHANGES_REQUESTED** [review]

**Pull request merged**